src/core/utils/fp64.js

```diff
@@ -28,6 +28,10 @@ export function fp64ify(a) {
   return [hiPart, loPart];
 }
 
+export function fp64LowPart(a) {
```

Comment:
Something to think about when using small math utils: bundlers, especially webpack, adds a thunk around every exported function that adds enough overhead to pull down performance, sometimes 2x. It may be worth it to also export these as part of an object:

```
export default {
   fp64LowPart,
}
```
  
You might actually be dodging the issue right now as this is reexported through the `experimental` object. If so, at least a short comment mentioning the issue at reexport time might be worth it.

For less frequently called methods, like `enable64bitSupport` below, this should of course not be done.